Gemfile

```
@@ -1,7 +1,7 @@
 source 'https://rubygems.org/'
 
 gemspec
-gem 'embulk'
+gem 'embulk', '< 0.10' # INFO: v0.9系のembulkコマンドの場合エラーになるのでバージョン指定してください。参考: https://zenn.dev/hiroysato/articles/957b1b4f77d549
```

コメントとバージョン指定の仕方が真逆？な気がします

0.9系が使えないようバージョン指定したいですね

```suggestion
gem 'embulk', '>= 0.10' # INFO: v0.9系のembulkコマンドの場合エラーになるのでバージョン指定してください。参考: https://zenn.dev/hiroysato/articles/957b1b4f77d549
```

---

0.9系のembulkコマンドを使いたい感じです！
troccoのworkerで動いているembulkが0.9.26なので

コメントを
```
INFO: v0.9系のembulkコマンドを使う場合はエラーになるのでバージョン指定してください。参考: https://zenn.dev/hiroysato/articles/957b1b4f77d549
```
のように修正しました。

---

0.9系を使うとエラーになる、という文章に見えるのですが、まだ良くわかってないですw

どのバージョン使うとエラーになる、みたいな書き方をした方がわかりやすいかもです

---

v0.9系のembulkコマンドで`embulk bundle`を実行するとエラーになります。
それを回避するために
```ruby
gem 'embulk', '< 0.10'
```
と指定する必要がある感じです。

---

↑ が伝えたいことになります。

---

修正しました。

---

https://github.com/trocco-io/embulk-output-bigquery/pull/23/commits/edea85693cec0526477b003ddb4e40bb9fa5aaed

---

ようやく理解しました。

0.9以下を使わないとエラーになるってことですね。

そもそもコメントいらない説ありますが（0.9系を使うのは見たらわかる）
書くとしたら `0.10系には非対応`とかですか？
あるいは、`0.10以上だとエラーが起きるため` とか

---

確かに、コメント書かなくても良いかもですねw
コメント消しときますね！

---

削除しました！
https://github.com/trocco-io/embulk-output-bigquery/pull/23/commits/ae9c662dee8425325c824aedad318d4d2bc789e3